### DISCUSSION TITLE
sudo password

### DISCUSSION THREAD

**Discussion by quadeqx** [dat-linux/community#46] [Issues]
Hello guys I have installed Dat Linux 1.0.2-lxqt and in the process, I forgot my sudo password. This means that I cannot install any applications using the terminal. Is there a way to reset it using the terminal? If there is or there is an alternative kindly assist me. Thanks in advance

**Comment by dat-linux** [comment]
Hi, try this: https://itsfoss.com/how-to-hack-ubuntu-password/

**Reply by quadeqx** [reply]
The process is useful however i have to reset it whenever I want to use sudo privileges on the command line
